src/store/sqlite_store/notes.rs

```diff
@@ -117,9 +131,24 @@ impl<'a> NoteFilter<'a> {
 
         match self {
             NoteFilter::All => base,
-            NoteFilter::Committed => format!("{base} WHERE status = 'Committed'"),
-            NoteFilter::Consumed => format!("{base} WHERE status = 'Consumed'"),
-            NoteFilter::Pending => format!("{base} WHERE status = 'Pending'"),
+            NoteFilter::Committed => {
+                format!(
+                    "{base} WHERE status = '{}' AND consumer_transaction_id IS NULL",
+                    NOTE_STATUS_COMMITTED
+                )
+            },
+            NoteFilter::Consumed => {
+                format!("{base} WHERE status = '{}'", NOTE_STATUS_CONSUMED)
```

Comment:
now that we use the constants we can just do:

```suggestion
                format!("{base} WHERE status = '{NOTE_STATUS_CONSUMED}'")
```

Reply:
Done!